src/views/clusteroverview/OverviewTab/getting-started-card/related-operators-section/RelatedOperatorsSection.tsx

```diff
@@ -14,6 +15,7 @@ const RelatedOperatorsSection: FC = () => {
   const { t } = useKubevirtTranslation();
   const { mtvLink, mtvLoaded } = useMTVResources();
   const isAdmin = useIsAdmin();
+  const [activeNamespace] = useActiveNamespace();
```

Comment:
`activeNamespace` can also have a special value that we have under the constant `ALL_NAMESPACES_SESSION_KEY` when the user have `all-namespaces` selected. You can use  `getValidNamespace` to make sure the string do not have that value.